components/tidb_query/src/codec/error.rs

```diff
@@ -1,16 +1,20 @@
 // Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
 
-use regex::Error as RegexpError;
-use serde_json::error::Error as SerdeError;
 use std::error::Error as StdError;
+use std::fmt::Display;
 use std::io;
 use std::num::ParseFloatError;
 use std::str::Utf8Error;
 use std::string::FromUtf8Error;
 use std::{error, str};
+
 use tipb::expression::ScalarFuncSig;
 use tipb::select;
 
```

Comment:
Remove this empty line.

Reply:
I think it is unnecessary, because the first part is std.

Reply:
I mean remove the L13 empty line, not L10.